src/ape_accounts/_cli.py

```diff
@@ -222,3 +224,38 @@ def delete(cli_ctx, alias):
     account = cli_ctx.account_manager.load(alias)
     account.delete()
     cli_ctx.logger.success(f"Account '{alias}' has been deleted")
+
+
+@cli.group()
+def auth():
```

Comment:
@antazoey what do you think of this?

Reply:
I guess one thing worth considering is "auth" often refers to "authentication" and not always "authorization", so if ever something closer to wallet-connect became a better thing, it might clash here